### PR TITLE
Send null instead of 'default' to Blink if post has default image 

### PR DESCRIPTION
**Pull request by chloealee**
#### What's this PR do?
If a post has a default image, send `null` instead of `default` to Blink. 

#### How should this be reviewed?
👀 

#### Relevant tickets
Fixes [this](https://www.pivotaltracker.com/n/projects/2019429/stories/152517900) Pivotal Ticket.

#### Checklist
- [ ] Documentation added for new features/changed endpoints.
- [ ] Tested on staging.
- [ ] Pinged a PM if this is a larger PR that would benefit from some additional testing love.

### CODE REVIEW

**Comment by chloealee** [comment]
cc @rapala61 @sergii-tkachenko

**Comment by sbsmith86** [comment]
Dope, I think we might also want to update our logic to [store this as `null` ](https://github.com/DoSomething/rogue/blob/master/app/Repositories/PostRepository.php#L73)  as well!

**Comment by chloealee** [comment]
@sbsmith86 create catch! 💡 

I updated this function in the `Three/PostRepository` file as well in [this PR](https://github.com/DoSomething/rogue/pull/526/files).

File: app/Models/Post.php [hunk 1]
```diff
@@ -102,8 +102,7 @@ public function tagSlugs()
     public function getMediaUrl()
     {
         if ($this->url === 'default') {
```
Comment (sbsmith86) [inline app/Models/Post.php]: Sorry wait! you will have to check for `null` here now as well!

**Comment by chloealee** [comment]
@sbsmith86 thanks for catching the other one! I did a search through the repo to make sure I didn't miss anymore and found one more in the front end! I think because of this, we'll need to change all the `default` values to `null` in the database too?

**Comment by sbsmith86** [comment]
@chloealee which component?

**Comment by sbsmith86** [comment]
I think for now we can just check for `null` and `default` for this column until we clean it up

File: resources/assets/helpers.js [hunk 2]
```diff
@@ -267,7 +267,7 @@ export function getImageUrlFromProp(photoProp) {
   }
 
 
-  if (photo_url == 'default') {
+  if (photo_url == 'default' || photo_url == null) {
```
Comment (chloealee) [inline resources/assets/helpers.js]: ah smart @sbsmith86 ! Updated that here in the helpers :)

**Pull request merged**